pkg/webhooks/validation/controller/controller.go

```diff
@@ -426,7 +429,7 @@ func (c *Controller) isDryRunOfInvalidConfigRejected() (rejected bool, reason st
 	if err == nil {
 		return false, fmt.Sprintf("dummy invalid config not rejected")
 	}
-	if !strings.Contains(err.Error(), deniedRequestMessageFragment) {
+	if !(strings.Contains(err.Error(), deniedRequestMessageFragment) || strings.Contains(err.Error(), missingResourceMessageFragment)) {
```

Comment:
Can you add a short comment with the context? 

It would also be helpful to log a warning when the resource type is missing.

Reply:
I added a comment + log